### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by simrin051**

### CODE REVIEW

**Review by srishtigautam21 — APPROVED** [review]
1.The size of pr is good
2.The code looks clean and readable
Good to merge

File: docs.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    .content-intro {
+        width: 80vw;
+        padding: 5px;
+        margin: 0px;
+
+    }
+
+    .content-display-container {
+        font-size: 15px;
+        width: 90vw !important;
```
Comment (parvez-rashid) [inline docs.css]: Using !important is a bad practice. If there are issues with values not getting applied, try to solve by manipulating specificity.

File: docs.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        padding: 5px;
+        margin: 0px;
+        margin-top: 5px;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+    }
+
+    iframe {
+        max-width: 90vw !important;
```
Comment (parvez-rashid) [inline docs.css]: Same as above, strictly avoid using !important.

File: docs.html [hunk 3]
```diff
@@ -0,0 +1,580 @@
...
+                <li><a href="#avatar">Avatar</a></li>
+                <li><a href="#badge">Badge</a></li>
+                <li><a href="#button">Button</a></li>
+                <li><a href="#card">Card</a></li>
+                <li><a href="#form">Form</a></li>
+                <li><a href="#image">Image</a></li>
+                <li><a href="#rating">Rating</a></li>
+                <li><a href="#toast">Toast</a></li>
+            </ul>
+        -->
```
Comment (parvez-rashid) [inline docs.html]: Remove comments before merging.

File: docs.html [hunk 4]
```diff
@@ -0,0 +1,580 @@
...
+                <div class="content-intro">
+                    <h1 class="content-title">Alert</h1>
+                    <p class="sub-content">
+                        Alert messages can be used to notify the user about something special: danger, success, information or warning.
+                    </p>
+                    <h1 class="content-title">Simple Alert</h1>
+                    <div class="content-display-container container-alerts">
+                        <div class="alert-box alert-success">
+                            <span class="alert-icon">
+                                <i class="fa fa-check" aria-hidden="true"></i>
```
Comment (parvez-rashid) [inline docs.html]: Great that you've used `aria-` attribute for better accessibility.

**Review by parvez-rashid — APPROVED** [review]
I've left a few comments, have a look. Overall, the code looks well-structured. Be careful about using the !important property (I've seen this in your quiz project as well) as it is to be avoided unless there is no other option. 

Code can be merged after making suggested changes.

**Pull request merged**